Accessories/uInverter/Design.md

```diff
@@ -0,0 +1,53 @@
+﻿## uInverter Design Calculations
+
+
+-   nominal dc link voltage - _12V_
+-   rated phase current - _2.5A rms_
+-   range of allowable switching frequency - _1kHz- 100kHz_
+-   range of allowable power frequency (phase frequency) - _640- 1000 Hz_
+
+ 
+### Core Components
+ #### DC Barrel Jack
+ 
+ The 2.1mm [DC barrel jack](https://www.digikey.com/en/products/detail/3643/1528-2553-ND/8346574) is rated for 15Vdc and is supplied from a standard [AC-DC wall adapter](digikey.com/en/products/detail/cui-inc/SWM30-12-NV-P5/7388285) rated for 12V. 
+
+#### DC link 
+A 3mF capacitor is selected as a DC link capacitor to protect the voltage ripple from input to the components comprising the half bridge. The capacitor is should be able to withstand a 12V input voltage. 
+
+Using formula 
+![](http://ec.kemet.com/wp-content/uploads/sites/4/2019/10/dc-link-design-tips-04-300x70.png)
+ * where _Vp(max)_ is chosen to be 0.5V
+ * duty cycle _(dc)_ assumed to be 0.75
+ * Switching frequency _fsw_ is 100kHz
+ * _Iout_ is a maximum of 7.5A 
+
+A 500mF Capacitor with a 1nF capacitor in parallel (to reduce the effect of ESR) are chosen for the DC link.
+
+#### Half Bridges
+
+The half bridge inverter is designed using [MAX5054 Gate Driver](https://datasheets.maximintegrated.com/en/ds/MAX5054-MAX5057.pdf) with an On-State maximum current of 4 A. This works well for a phase current chosen as 2.5Arms. 
+
+The gate driver uses the 12V input voltage as VDD, which fits in the defined range of 4V-15V. The protection circuit comprises 0.1uF capacitor, 1ohm resistor and a protection diode on the input side and a 1uF capacitor between Vdd and ground pins. 
+
+
+#### R-L Load and Current Sense Resistor
+
+Each phase of the inverter is connected to an R-L load and a sense resistor. Since each phase receives 50% of the bus voltage, i.e., 6V and 2.5Arms as regulated by the driver used in the half-bridge. 
+
+Assuming phase frequency is 1kHz, a 320uH inductor and a 400mOhm resistor is chosen, where the resistor doubles up as a current sense resistor. The 400 mOhm value of the resistor will be justified in the next section, and the impedance of the inductor (Z= 2* *pi* **f* *L) is 2 ohm. In total, an impedance of 2.4ohm draws a maximum of 3.53A in each phase (set at 6V). 
```

Comment:
I think the outcome of this section is to size the inductor for the RL load, which you state that 320uH will work. Please make this more clear that this the outcome.

Furthermore, in this section, you state that 3.53A is the maximum current. This is correct at 1kHz frequency due to RL impedance, but at lower frequencies, MUCH larger current could flow in the RL load.